pushapkscript/apk.py

```diff
@@ -7,16 +7,21 @@
 _DIRECTORY_WITH_ARCHITECTURE_METADATA = 'lib/'     # For instance: lib/x86/ or lib/armeabi-v7a/
 _ARCHITECTURE_SUBDIRECTORY_INDEX = len(_DIRECTORY_WITH_ARCHITECTURE_METADATA.split('/')) - 1    # Removes last trailing slash
 
-_EXPECTED_MOZAPKPUBLISHER_ARCHITECTURES = ('armv7_v15', 'x86')
+_EXPECTED_MOZAPKPUBLISHER_ARCHITECTURES_PER_CHANNEL = {
+    # XXX arm64-v8a to come in Aurora (Bug 1368484)
+    'aurora': ('armv7_v15', 'x86'),
+    'beta': ('armv7_v15', 'x86'),
+    'release': ('armv7_v15', 'x86'),
+}
 
 
-def sort_and_check_apks_per_architectures(apks_paths):
+def sort_and_check_apks_per_architectures(apks_paths, channel='release'):
```

Comment:
I'd lean towards not having a default channel... explicitly passing seems more precise.